prestoadmin/util/presto_config.py

```diff
@@ -52,7 +51,7 @@ class PrestoConfig:
         LDAP_CLIENT_PASSWORD_KEY: None
     }
 
-    def __init__(self, config_properties, config_host, config_path):
+    def __init__(self, config_properties, config_path, config_host):
```

Comment:
Can you change the commit message to "change order of arguments for PrestoConfig"